server/src/main/java/org/elasticsearch/rest/MethodHandlers.java

```diff
@@ -22,26 +22,21 @@
     private final String path;
     private final Map<RestRequest.Method, Map<RestApiVersion, RestHandler>> methodHandlers;
 
-    MethodHandlers(String path, RestHandler handler, RestRequest.Method... methods) {
+    MethodHandlers(String path) {
         this.path = path;
-        this.methodHandlers = new HashMap<>(methods.length);
-        for (RestRequest.Method method : methods) {
-            methodHandlers.computeIfAbsent(method, k -> new HashMap<>())
-                .put(handler.compatibleWithVersion(), handler);
-        }
+        this.methodHandlers = new HashMap<>(4);
```

Comment:
can you add a comment about the choice of `4` for the initial map size?

Reply:
Good call, 9afd325cd756b318137f51804badf52cc8e962db.